en/advanced-topics/meta.md

```diff
@@ -1,15 +1,15 @@
-# Resource Management Considerations --- meta files
+**meta** files# Resource Management Considerations --- meta files
```

Comment:
前面多了：**meta** files

Reply:
Thanks I don't know how that keeps happeneing. Every document seems to end up with text at the frist line before the __#__. I'll try to figure this out because when I test it isn't showing up. I'm using Atom. What editor are you using?

Reply:
I use VS Code to edit, and it can preview in editor,like this:
![test](https://user-images.githubusercontent.com/19702899/49558988-06b09f00-f948-11e8-8cde-287d453c2072.png)

Or you can specify to the creator-docs directory, using the command line to quickly preview:
```bash
npm run preview -- -o file(such as: mata)
```